index.html

```diff
@@ -222,7 +229,8 @@
         <div class='small-12 columns'>
           <h2>Any Questions?</h2>
           <p>
-            Please reach out! We can be reached anytime at <a class="email" href="mailto:summer@hackclub.com">summer@hackclub.com</a>.
+            Please reach out! We can be reached anytime at <a class="email" href="mailto:summer@hackclub.com">summer@hackclub.com</a>.<br>
+             © 2016 <a class="email" href="https://hackclub.com">Hack Club</a>, a 501(c)(3) non-profit [Tax ID46-0945785].
```

Comment:
The tax id section should have a space after `ID`. So it should be `[Tax ID 46-0945785]`.